blazor/chart/strip-line.md

```diff
@@ -199,6 +199,52 @@ The [Start](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Charts.Chart
 
 ![Blazor Chart with Custom Stripline Text](images/strip-line/blazor-chart-custom-strip-text.png)
 
+## Segemented Stripline
```

Comment:
Can you check the ej2 and add like this.
https://ej2.syncfusion.com/angular/documentation/chart/strip-line#segment-stripline

Reply:
resloved